Library/Formula/websockify.rb

```diff
@@ -0,0 +1,71 @@
+class Websockify < Formula
+  homepage "https://github.com/kanaka/websockify"
+  url "https://github.com/kanaka/websockify/archive/v0.6.0.tar.gz"
+  sha1 "f2bffebc9e4d94ad3a329638bee63e899eaad9c0"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on "openssl"
+
+  def install
+    system "make"
+
+    Dir.chdir("other") do
+      system "make"
+    end
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    # mv this so it is not installed with the other docs
+    mv "docs/websockify.1", "websockify.1"
+    man1.install "websockify.1"
+    (doc+"websockify").install "docs"
+
+    # Upstream Bug: Not named ".dylib"
+    lib.install "rebind.so"
```

Comment:
Can you link that in the formula file and ideally also create a pull request upstream? Thanks!